.github/workflows/deploy_pr_on_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger Deploy on Command
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  deploy-test:
+    if: startsWith(github.event.comment.body, '/deploy')
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Trigger deployment workflow
+        uses: peter-evans/slash-command-dispatch@v2
```

Comment:
An interesting action, but I'm interested to know what advantage does this provide issuing a workflow_dispatch in Actions?

Reply:
An advantage compared to what? The idea is that this allows the test deploy to be triggered by calling /deploy in a PR. Based on my understanding, this is the easiest way to do it ...

Reply:
Perhaps I'm missing something here, I can see this will allow a `/deploy` command to be used in PRs.

However, can you not just run your feature branch from https://github.com/forrtproject/forrtproject.github.io/actions/workflows/deploy.yaml by pressing 'Run Workflow' and selecting the branch you want to execute it against? That is what I'm trying to compare against. Assuming the `deploy.yml` is not modified in the PR, the workflow will still checkout the feature branch you selected.

Reply:
Sure, that should work - it's just more complicated than comments on the PR. Do you have any concerns about this action?

Reply:
It is usually good practice to avoid 3rd party-actions when necessary, in line with guidance which you can find here https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions. 

I'm not suggesting this action is malicious, but its just something to bare in mind. Imo remembering commands can be more dauting for inexperienced contributors than clicking a button. Whatever way this is done, I suggest updating the `CONTRIBUTING.md` page to reflect this.

Reply:
/remind me to respond tomorrow [mostly just testing slash command]

Reply:
Here I already tricked myself - if we keep this, it also needs to run on `pull_request_review_comment`

Reply:
I see we can use a newer version of this workflow. Could you please we raise this ChatOps workflow in a separate PR so we can separate the staging environment from this.

https://github.com/peter-evans/slash-command-dispatch/tree/v4/

Reply:
You mean to remove this from this PR? I can't do that today, but feel free to do so - otherwise I will do it asap. 

Thanks for your other work on this!

Reply:
Ok, I saw that you already removed it. Thanks! Will do the new PR next week. 

/remind me to raise `/deploy` PR